### DISCUSSION TITLE
How to obtain the confirmed bugs of OpenSSL from the past 10 years?

### DISCUSSION THREAD

**Discussion by onepeople158** [openssl/openssl#26137] [Q&A]
Hello Developer,

I would like to obtain the confirmed bugs of OpenSSL from the past few years. Could you please let me know which websites or other methods I can use to query the confirmed bugs from the recent years?

**Comment by paulidale** [comment]
`git log`

**Reply by onepeople158** [reply]
> `git log`

Hello developer, I tried to use the git log command, so I ran git clone https://github.com.cnpmjs.org/openssl/openssl.git, but I received an error message.
![屏幕截图 2024-12-09 172725](https://github.com/user-attachments/assets/37958f38-2ef5-4be1-90f7-22da2b629b04)
Could you please tell me what the issue might be and how I should fix it?

**Reply by paulidale** [reply]
Try: https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account
Or clone https://github.com/openssl/openssl.git

**Reply by levitte** [reply]
"Failed to connect to github.cin port 443" suggests a connection problem.  Have you tried checking the network to see what that's about?  You know, simple commands like:

`ping github.com`

`traceroute github.com`

`curl -Lv github.com`

`ssh -v github.com`

**Comment by tom-cosgrove-arm** [comment]
> I ran git clone https://github.com.cnpmjs.org/openssl/openssl.git

Doesn't look right - should be

```
$ git clone https://github.com/openssl/openssl.git           
Cloning into 'openssl'...
remote: Enumerating objects: 494116, done.
remote: Counting objects: 100% (18302/18302), done.
remote: Compressing objects: 100% (923/923), done.
:
```

Check you can visit https://github.com/ in your browser.

https://github.com/openssl/openssl/pulls?q=is%3Apr+is%3Aclosed+label%3A%22triaged%3A+bug%22 will show you many of the PRs that fixed bugs, although not all.
